ledger_device_sdk/src/ui/gadgets.rs

```diff
@@ -66,6 +66,31 @@ pub fn clear_screen() {
     BLANK.paint();
 }
 
+// Developer mode / pending review popup
+// must be cleared with user interaction
+pub fn display_pending_review(comm: &mut Comm) {
```

Comment:
Are we sure we want this function defined in the gadgets module ?

Reply:
Where do you think it would fit better?

Reply:
Aside the `popup` method seems the most logical at the moment.